api/v1/Game.php

```diff
@@ -674,6 +674,7 @@ public function GetGameDetails()/*: array|PromiseInterface // <-- php 8 */
             }
 
             $realtimePerEra = explode(",", $state["game_planning_era_realtime"]);
+            // todo: division by zero.
```

Comment:
Good point. Looks like the client has a validation built in to ensure these values are at least one. I couldn't find a function in the Game class (notably Realtime() or FutureRealtime()) that validated this. And the database schema does allow for NULL values on the game_planning_realtime column (although the default is 1). Something to think about, but not necessary for beta10 I would say indeed.